lang/java/wiredtiger.i

```diff
@@ -2274,7 +2273,7 @@ err:		if (ret != 0)
 			cursor->flags |= WT_CURSTD_RAW;
 
 		jcb->jnienv = jenv;
-		jcb->session = (WT_SESSION_IMPL *)cursor->session;
+		jcb->session = CUR2S(cursor);
```

Comment:
Were these missed when CUR2S was added? Why are they here?

Reply:
There seem to be a few more places where you are switching in the new macro, but that aren't related to this change - does it make sense to split that change out?
I think we want to revisit this change at some stage to make sure that the right btree is in the session handle for modify to make the decision without the parameter - having this patch as clean as possible will make that easier.

Reply:
I'll clean that in a separate ticket.

Reply:
Created WT-6109